packages/valory/skills/market_creation_manager_abci/behaviours.py

```diff
@@ -1399,8 +1397,8 @@ def _get_withdraw_tx(self, amount: int) -> Generator[None, None, Optional[Dict]]
         response = yield from self.get_contract_api_response(
             performative=ContractApiMessage.Performative.GET_STATE,
             contract_address=self.params.conditional_tokens_contract,
-            contract_id=str(WxDAIContract.contract_id),
-            contract_callable=get_callable_name(WxDAIContract.build_withdraw_tx),
+            contract_id=str(ERC20.contract_id),
+            contract_callable=get_callable_name(ERC20.build_withdraw_tx),
```

Comment:
The output format of `_get_withdraw_tx` (line 1409) does not need to be modified, even with the difference in interface between former WxDAI contract and ERC20 contract. This is because, downstream, the multisend contract correctly works for "0x" strings and bytes.